### DISCUSSION TITLE
TODO List

### DISCUSSION THREAD

**Discussion by bedovyy** [bedovyy/chibi-client#1] [General]
- [ ] add lora selector
- [ ] DataManager refactoring
- [ ] check side effect of save as webp (which uses SaveAnimatedWEBP node)
- [ ] support Stable Diffusion web UI scheduler (1.9.0~)
- [x] support Stable Diffusion web UI
- [x] add resolution preset
- [x] progress loading of history
- [x] improve initialization performance

**Comment by 30016654** [comment]
Hi there,  do you have an estimated time for lora selection options?  I love your front end and the only missing element is the ability to select loras.

**Reply by bedovyy** [reply]
okay, i will add lora soon.

**Reply by 30016654** [reply]
If possible the ability to use multiple loras and slider koras would be amazing

**Comment by usermx01** [comment]
Great gui you made for working on mobile. Any chance to add support for flux loras as well or inpainting or controlnets? I seen sort of this function in 1min.ai site and wonder of you see your tool could go in that kind of direction for image generation functionality based on comfyui.
